index.js

```diff
@@ -4,8 +4,7 @@ const defaultBrowsers = [
   'last 3 Safari versions',
   'last 4 Edge versions',
   'Firefox ESR',
-  'IE 11',
-  'Firefox >= 45'
+  'Chrome >= 55'
```

Comment:
Should `Chrome >= 55` be here if we have `last 8 Chrome versions` at the top?

Reply:
Just mimicking our internal `.browserlistrc`